wsd/models/__init__.py

```diff
@@ -1,2 +1,5 @@
 """A collection of WSD models."""
-from .baseline import *
+# from .baseline import *
```

Comment:
Not the right imports here I think. Baseline should get a few more like `JMDict`.